frontend/src/components/CodeEditor.tsx

```diff
@@ -62,21 +62,21 @@ function CodeEditor(): JSX.Element {
           <Tab
             key={
               selectedFileName === ""
```

Comment:
Can we replace this conditional with just `selectedFileName.toLocaleLowerCase()`?